cmd/sqlflowserver/main_test.go

```diff
@@ -736,3 +739,52 @@ LABEL "label" INTO model_table;`, caseDB)
 	// wait train finish
 	ParseRow(stream)
 }
+
+func TestSQLByPass(t *testing.T) {
+	testDBDriver := os.Getenv("SQLFLOW_TEST_DB")
+	// default run mysql tests
+	if testDBDriver != "mysql" {
+		t.Skip("Skipping mysql tests")
+	}
+	dbConnStr = "mysql://root:root@tcp/?maxAllowedPacket=0"
+	modelDir := ""
+
+	tmpDir, caCrt, caKey, err := generateTempCA()
+	defer os.RemoveAll(tmpDir)
+	if err != nil {
+		t.Fatalf("failed to generate CA pair %v", err)
+	}
+	go start("", modelDir, caCrt, caKey, true)
```

Comment:
`TestEnd2EndMySQL` also starts an SQLFlow server.

Reply:
Thanks for fixes!